Main.py

```diff
@@ -0,0 +1,21 @@
+from Battery_check import battery_is_ok
+
+if __name__ == '__main__':
+
+  temp_min = 0
+  temp_max = 45
+  soc_min = 20
+  soc_max = 80
+  charge_rate_min = None
+  charge_rate_max = 0.8
+
+
+  assert(battery_is_ok(25, 70, 0.7,temp_min,temp_max,soc_min,soc_max,charge_rate_min,charge_rate_max) is True)
```

Comment:
The user of the API now needs to also know the min and max values of each parameter. Please think over if that is how it should be...